### PR TITLE
Added Tryhackme Site

### PR DESCRIPTION
**Pull request by whiskerwind**

### CODE REVIEW

File: entries/t/tryhackme.com.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "Try Hack Me": {
```
Comment (ApCoder123) [inline entries/t/tryhackme.com.json]:
```suggestion
  "TryHackMe": {
```

The name doesn't have any spaces in between on the site.

**Review by ApCoder123 — CHANGES_REQUESTED** [review]

**Comment by phallobst** [comment]
How about this SVG icon:

**Comment by whiskerwind** [comment]
Good idea on the logomark instead of the logotype

File: entries/t/tryhackme.com.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "TryHackMe": {
+    "domain": "tryhackme.com",
+    "contact": {
+     "twitter": "RealTryHackMe",
+     "email": "hello@tryhackme.com"
+    },
+    "keywords": [
+      "education"
```
Comment (phallobst) [inline entries/t/tryhackme.com.json:10]:
```suggestion
      "security",
      "education"
```

**Review by phallobst — CHANGES_REQUESTED** [review]

**Review by phallobst — APPROVED** [review]

**Review by ApCoder123 — APPROVED** [review]

**Pull request merged**